test/integration/helpers/stake/Facilitator.js

```diff
@@ -45,58 +43,60 @@ const valueTokenInWei = '200',
   gasPrice = '8000000',
   gasLimit = '100';
 
-let txOptions = {
-  from: owner,
-  gas: '8000000'
-};
-
 describe('Facilitator', async function() {
-  let deployParams = {
-    from: config.deployerAddress,
-    gasPrice: config.gasPrice
-  };
-
-  before(function() {
-    this.timeout(3 * 60000);
+  let deployerAddress;
+  let deployParams;
+  let txOptions;
+
+  before(async function() {
+    // Set up docker geth instance and retrieve RPC endpoint
+    const { rpcEndpointOrigin } = await dockerSetup();
+    web3 = new Web3(rpcEndpointOrigin);
+    const accountsOrigin = await web3.eth.getAccounts();
+    deployerAddress = accountsOrigin[0];
+    owner = deployerAddress;
+    deployParams = {
+      from: deployerAddress,
+      gasPrice: config.gasPrice
+    };
 
-    return web3WalletHelper
-      .init(web3)
-      .then(function(_out) {
-        if (!caOrganization) {
-          console.log('* Setting up Organization');
-          wallets = web3WalletHelper.web3Object.eth.accounts.wallet;
-          worker = wallets[1].address;
-          beneficiary = wallets[2].address;
-          facilitator = wallets[3].address;
-          let orgHelper = new OrganizationHelper(web3, caOrganization);
-          const orgConfig = {
-            deployer: config.deployerAddress,
-            owner: owner,
-            workers: worker,
-            workerExpirationHeight: '20000000'
-          };
-          return orgHelper.setup(orgConfig).then(function() {
-            caOrganization = orgHelper.address;
-          });
-        }
-        return _out;
-      })
-      .then(function() {
-        if (!caMockToken) {
-          deployer = new MockContractsDeployer(config.deployerAddress, web3);
-          return deployer.deployMockToken().then(function() {
-            caMockToken = deployer.addresses.MockToken;
-            return caMockToken;
-          });
-        }
+    beneficiary = accountsOrigin[2];
+    facilitator = accountsOrigin[1];
```

Comment:
Although only 3 addresses are unlocked, [more than 3 are funded](https://github.com/AugurProject/ethereum-nodes/blob/master/geth-poa/genesis.json). Presumably, to use the other funded addresses, one simply needs to `unlock` them. I did consider that here, but concluded it wasn't essential to the task.

If it is preferred to have more addresses unlocked from the start, I think we would need to change the docker image.

Reply:
Not needed more than 3 for now.
We will enable later based on requirement. It's good to be aware that we have the flexibility to that. 
May I know how can we configure how many addresses to unlock?

Reply:
It looks like web3 only lets you unlock one at a time. But maybe the code has already been written in-house for handling unlocking in a bulkier way for previous interaction layers?

Given some of your questions, in a cooler moment, it might make sense to investigate making a docker image that would provide some more flexibility and convenience.